### PR TITLE
[Cognitive-language] Fixed cspell typos in QuestionAnswering

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# Description

fix https://github.com/Azure/azure-sdk-for-python/issues/22595

Hello @kristapratico! I Fixed the spelling errors inside QuestionAnswering, and I made sure to not overfill the words in cspell.json. May you please review it? Thank you!

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def test_query_text_llc(self, qna_account, qna_key):
                     "id": "doc2"
                 },
                 {
+                    # cspell:ignore groovin Olufsen
```
Comment (kristapratico) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py:41]: We try not to add cspell ignores inline with the code. Would you mind adding these words to the cspell.json override section under questionanswering? Thanks!

**Review by kristapratico — COMMENTED** [review]
Thank you so much! Looking good, only one small thing.

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py (hunk 1 above)
Comment (wonhyeongseo) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py]: As you wish, madame! Thank you for showing me best practices; it's such a valuable asset for a beginner like me. Bless you and stay healthy Krista!

Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py]: No problem! Thanks for making the changes! I pushed a commit to fix the merge conflict, once that's green in CI I will approve and merge! 😸

**Review by kristapratico — APPROVED** [review]

**Pull request merged**